core/arch/arm/pta/benchmark.c

```diff
@@ -62,43 +69,61 @@ static TEE_Result register_benchmark_memref(uint32_t type,
 {
 	TEE_Result res;
 
-	if ((TEE_PARAM_TYPE_GET(type, 0) != TEE_PARAM_TYPE_MEMREF_INOUT) ||
+	if ((TEE_PARAM_TYPE_GET(type, 0) != TEE_PARAM_TYPE_VALUE_INOUT) ||
```

Comment:
Based on how the function has been changed, it should maybe be renamed from `register_...` to `allocate_...`?

Reply:
@jbech-linaro will fix

Reply:
fixed